lib/SILOptimizer/Mandatory/TFUtilities.cpp

```diff
@@ -372,6 +385,14 @@ bool TensorFunctionClassifier::shouldBePartitioned(SILFunction *fn,
     }
   }
 
+  // If this is a function that was inlined from some other module but only
```

Comment:
The comment suggests that this check should happen only for functions that were inlined. However, we violate that invariant after the refactor.  Can you update the comment accordingly?

Reply:
The comment did not suggest so. 

Before this patch, this check runs only on functions that are inlinable (if a func were `@inline(never)`, it would have returned by getting to this point). After this patch, it could run even on functions that are `@inline(never)`, such as `_hostOp` in the PR description. The latter is the correct behavior -- we want to return false when we see such an "out of module" function, regardless of whether it is inlinable.

I updated this comment here to make things more clear.